src/main.ts

```diff
@@ -335,6 +336,21 @@ export const fold = <E, A, B>(
   }
 };
 
+export function getEq<A>(E: Eq<A>): Eq<Dataway<unknown, A>> {
```

Comment:
question ici: est-ce qu'on veut comparer les cas de failure aussi ?
Dans ce cas il faut ça : 
```suggestion
export function getEq<E, A>(EE: Eq<E>, EA: Eq<A>): Eq<Dataway<E, A>> {
```

Reply:
Je pense que non car on veut généralement comparer les données qu'on est aller fetch, pas les erreurs

Reply:
Du coup je suis passé sur EE et EA pour suivre la philo fp-ts